db/migrate/20160211134035_remove_system_errata.rb

```diff
@@ -0,0 +1,33 @@
+class RemoveSystemErrata < ActiveRecord::Migration
+  class System < ActiveRecord::Base
+    self.table_name = "katello_systems"
+
+    has_many :system_errata, :class_name => "Katello::SystemErratum", :dependent => :destroy, :inverse_of => :system
+  end
+
+  class SystemErratum < ActiveRecord::Base
+    self.table_name = "katello_system_errata"
+    belongs_to :system, :inverse_of => :system_errata, :class_name => 'Katello::System'
+
+  end
```

Comment:
You don't actually need these class definitions if you aren't referencing the classes below.